Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -3,156 +3,138 @@ import SnowplowTracker
 import Core
 
 final class Analytics {
+    private static var tracker: TrackerController {
+        Snowplow
+            .createTracker(namespace: "ios_sp",
+                           network: .init(endpoint: Environment.current.snowplow),
+                           configurations: [TrackerConfiguration()
+                                                .appId(Bundle.version)
+                                                .platformContext(true)
+                                                .geoLocationContext(true),
+                                            SubjectConfiguration()
+                                                .userId(User.shared.analyticsId.uuidString)])
+    }
+    
     static let shared = Analytics()
-    private let tracker: SPTracker
+    private(set) var tracker: TrackerController
 
     private init() {
-        tracker = .build {
-            $0?.setAppId(Bundle.version)
-            $0?.setTrackerNamespace("ios_sp")
-            
-            $0?.setEmitter(.build {
-                $0?.setUrlEndpoint(Environment.current.snowplow)
-            })
-            
-            let subject = SPSubject(platformContext: true, andGeoContext: true)
-            subject?.setUserId(User.shared.analyticsId.uuidString)
-            $0?.setSubject(subject)
-        }
+        tracker = Self.tracker
     }
     
     func install() {
-        SPSelfDescribingJson(schema: "iglu:org.ecosia/ios_install_event/jsonschema/1-0-0", andData: ["app_v": Bundle.version] as NSObject).map { data in
-            tracker.track(SPUnstructured.build {
-                $0.setEventData(data)
-            })
-        }
+        tracker
+            .track(SelfDescribing(schema: "iglu:com.snowplowanalytics.snowplow/link_click/jsonschema/1-0-1",
```

Comment:
maybe we can remove this if `installAutotracking ` is enabled? otherwise it could turn out we count installs twice if auto tracking uses the same schema (what might be)

Reply:
@ecotopian As far as I remember we needed to set our own install event as on the BI side they could not read it directly from the auto tracked one. I remember a conversation around it last year, but there is no documentation and better don't take my word for it. This should also be on a separated _optimisation_ ticket.